config/routes.rb

```diff
@@ -4,11 +4,17 @@
   root to: redirect('/documents')
   get '/dev' => 'development#index'
 
+  get '/documents/new' => 'new_document#choose_supertype', as: :new_document
+  post '/documents/new' => 'new_document#choose_document_type'
+  post '/documents/create' => 'new_document#create', as: :create_document
```

Comment:
Good points both. I wrote about this in https://github.com/alphagov/content-publisher/pull/26/commits/7b1d315af7d57943a0b9cd30510bdde3d4d26da0 but maybe it got missed:

> I've pulled out the choosing format into its own controller. This probably makes sense, because everything will be related to documents in this application. If we follow a REST-ish method we'll end up with a huge controller.

I definitely think we should clean up the routes, and perhaps could use `resources`, but for now making them super explicit and duplicated allows us to iterate quickly. At this point, I'm not sure that REST applies to this application and we're not better off modelling it otherwise.